src/components/upload/UploadArea.tsx

```
@@ -0,0 +1,101 @@
+        onDragLeave={handleDragLeave}
+        onDrop={handleDrop}
+        onClick={() => fileInputRef.current?.click()}
+        onKeyDown={(e) => { if (e.key === 'Enter' || e.key === ' ') { e.preventDefault(); fileInputRef.current?.click(); } }}
+        role="button"
+        tabIndex={0}
+        className={`relative border-2 border-dashed rounded-xl p-8 text-center cursor-pointer
+          ${dragOver ? 'border-teal-500 bg-teal-500/10' : 'border-dark-700 hover:border-teal-500/50 bg-dark-900/50'}`}
+      >
+        <input
+          ref={fileInputRef}
+          type="file"
```

Biome (2.3.14)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)